src/components/Table/types.ts

```diff
@@ -37,12 +37,28 @@ export type columnTypes = Columns;
 /** The data for a single row. */
 export type RowEntry = Record<string, any>;
 
+export interface SortState {
+  /** The key of the header column that is currently being sorted by. */
+  sortedColumn?: string;
+  direction: SortDirection;
+}
+
+export enum SortDirection {
+  noSort = 0,
+  ascending = 1,
+  descending = 2,
+}
```

Comment:
I believe enums do this automatically
```suggestion
export enum SortDirection {
  noSort,
  ascending,
  descending,
}
```